src/ffi/mod.rs

```diff
@@ -929,7 +1012,17 @@ impl Subscriber for TelioTracingSubscriber {
         };
         event.record(&mut visitor);
 
-        if let Some(filtered_msg) = filter_log_message(visitor.message) {
+        #[cfg(feature = "hide_ips")]
+        let message = if let Some(censor) = self.censor.as_ref() {
+            censor.censor_logs(visitor.message)
+        } else {
+            visitor.message
+        };
+
+        #[cfg(not(feature = "hide_ips"))]
+        let message = visitor.message;
+
+        if let Some(filtered_msg) = filter_log_message(message) {
             let _ = self.callback.log(level.into(), filtered_msg);
```

Comment:
I think that modification of logs would be best done here - to avoid needlessly modifying logs that will not be printed.